presto-main/src/test/java/io/prestosql/operator/aggregation/minmaxby/TestMinMaxByAggregation.java

```diff
@@ -74,23 +79,21 @@ public void testAllRegistered()
         return new ImmutableList.Builder<Type>()
                 .addAll(simpleTypes)
                 .add(VARCHAR)
-                .add(DecimalType.createDecimalType(1))
+                .add(createDecimalType(1))
                 .add(RowType.anonymous(ImmutableList.of(BIGINT, VARCHAR, DOUBLE)))
                 .build();
     }
 
     @Test
     public void testMinUnknown()
     {
-        InternalAggregationFunction unknownKey = METADATA.getFunctionManager().getAggregateFunctionImplementation(
-                new Signature("min_by", AGGREGATE, parseTypeSignature(UnknownType.NAME), parseTypeSignature(UnknownType.NAME), parseTypeSignature(StandardTypes.DOUBLE)));
+        InternalAggregationFunction unknownKey = getInternalAggregationFunction("min_by", UNKNOWN, DOUBLE);
         assertAggregation(
                 unknownKey,
                 null,
                 createBooleansBlock(null, null),
                 createDoublesBlock(1.0, 2.0));
-        InternalAggregationFunction unknownValue = METADATA.getFunctionManager().getAggregateFunctionImplementation(
-                new Signature("min_by", AGGREGATE, parseTypeSignature(StandardTypes.DOUBLE), parseTypeSignature(StandardTypes.DOUBLE), parseTypeSignature(UnknownType.NAME)));
+        InternalAggregationFunction unknownValue = getInternalAggregationFunction("min_by", DOUBLE, UNKNOWN);
         assertAggregation(
                 unknownKey,
```

Comment:
I'm guessing this is meant to be `unknownValue`, otherwise `unknownValue` is an unused variable.